settings.py

```diff
@@ -1396,6 +1398,13 @@
 if 'corehq.sql_db.routers.MultiDBRouter' not in DATABASE_ROUTERS:
     DATABASE_ROUTERS.append('corehq.sql_db.routers.MultiDBRouter')
 
+# Mapping of app_label to DB name or reporting DB alias (see REPORTING_DATABASES)
+CUSTOM_DB_ROUTING = {
```

Comment:
Isn't the plan to do this via commcare-cloud setting override?

Reply:
https://github.com/dimagi/commcare-cloud/pull/3972/files#diff-6604a8eb62ac7594d06f38182d65ffa2R374-R375